ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -72,25 +72,28 @@ resource "azurerm_monitor_smart_detector_alert_rule" "failure_anomalies" {
 
 resource "azurerm_monitor_scheduled_query_rules_alert" "http_2xx_failed_requests" {
   name                = "${var.env}-api-2xx-failed-requests"
-  description         = "${local.env_title} HTTP Server 2xx Errors (where successful request == false) >= 25"
+  description         = "${local.env_title} HTTP Server 2xx (requests where (failed requests * 100.00 / total requests) >= ${var.http_2xx_failure_rate_threshold}). This query finds all requests in our timeframe and transforms the failed requests into a percentage of total requests."
   location            = data.azurerm_resource_group.app.location
   resource_group_name = var.rg_name
   severity            = var.severity
   frequency           = 5
   time_window         = 5
-  enabled             = contains(var.disabled_alerts, "http_2xx_failed_requests") ? false : true
```

Comment:
why remove the disabled_alerts check?

Reply:
That was silly of me, I just did that for testing. It's back!